conneg-by-ap/index.html

```diff
@@ -1157,16 +1157,43 @@ <h4>HTTP Alternate Representations Data Model Implementation</h4>
         <section id="listprofiles-tokens">
           <h3>Token mappings</h3>
           <p>
-            If HTTP <a>server</a>s wish to allow <a>client</a>s to identify <a>profile</a>s via <a>token</a>,
+            If HTTP <a>server</a>s wish to allow <a>client</a>s to identify <a>profile</a>s via <a>token</a>s,
             in addition to the the mandatory identification of <a>profile</a>s via via URI, the <a>server</a>s will
-            need to provide a <a>token</a> / URI mapping via this <strong>list profiles</strong> function. In this
-            HTTP functional profile of this specification, this may be implemented in a number of ways. The preferred
-            way is to add "token" parameters to the HTTP Content-Profile and Link HTTP headers.
+            need to provide a <a>token</a> / URI mapping via this <strong>list profiles</strong> function.
```

Comment:
We _could_ write this instead as "needs to indicate for every supported token to which profile URI it corresponds". This eliminates the concept of a "token / URI mapping" (and associated questions such as whether it needs to be complete, injective, etc.).